app/default/data.py

```diff
@@ -210,7 +210,7 @@ def get_application_display_config(fund_id, round_id, language):
     try:
         return [ApplicationMapping.from_dict(section) for section in application_display_response]
     except Exception as e:
-        raise ValueError(f"Failed to create ApplicationMapping instance: {str(e)}")
+        raise ValueError("Failed to create ApplicationMapping instance") from e
```

Comment:
Do we want to include the error or some of its info in the ValueError being raised?

Reply:
The info from the previous error gets chained as context into this new error via the `from e`, so the context should be visible from the stacktrace still.